package.json

```diff
@@ -17,7 +18,9 @@
     "package": "cross-env NODE_ENV=production node -r babel-register -r babel-polyfill webpack/package.js",
     "package-all": "npm run package -- --all",
     "postinstall": "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json",
+    "preinstall": "which electron || npm install electron@1.4.12",
```

Comment:
Im still against putting a version number here … this is not expected and will lead to confusion down the road (e.g: nix electron version might be different than this one). It's easy to forget to update it here. The only option i see is to put the electron version in a config file that can be loaded in both environments or behind an environment flag.